tests/bayesian/dropconnect_test.py

```diff
@@ -87,22 +57,5 @@ def test_patch_module_raise_warnings(inplace, layers):
         assert "No layer was modified by patch_module" in str(w[-1].message)
 
 
-def test_weight_change_after_forward_pass():
```

Comment:
why dont we test these anymore?

Reply:
We don't keep a copy of the weights anymore. So there is not much to test. I could mock `conv_forward` and `F.linear` to test it.